client/components/domains/featured-domain-suggestions/style.scss

```
@@ -118,7 +118,7 @@
 	}
 
 	.domain-registration-suggestion__badges {
-		align-self: flex-start;
+		align-self: center;
 		align-items: center;
```

I think if we use this:

```suggestion
		align-self: flex-start;
		align-items: flex-end;
```

...then it should also take care of the vertical alignment case mentioned in https://github.com/Automattic/nomado-issues/issues/616 as well.

---

@delputnam `align-self: center;` works in both cases, I've tested it.

The other issue - for the horizontal space part - is related with word break and flex grow. It could be fixed using this rule on H3 element (not supported in FF yet).
`text-wrap:pretty`